packages/react-textarea/src/stories/TextAreaUncontrolled.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { TextArea, TextAreaProps } from '../index';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles, shorthands } from '@griffel/react';
+
+const useStyles = makeStyles({
+  container: {
+    '> div': { ...shorthands.padding('20px') },
+    '& label': { display: 'block', marginBottom: '10px' },
+  },
+});
+
+const onChange: TextAreaProps['onChange'] = (ev, data) => {
+  // Uncontrolled inputs can be notified of changes to the value
+  console.log(`New value: "${data.value}"`);
+};
+
+export const Uncontrolled = () => {
+  const textareaId = useId('textarea');
+  const styles = useStyles();
+
+  return (
+    <div className={styles.container}>
+      <Label htmlFor={textareaId}>Uncontrolled TextArea with placeholder.</Label>
+      <TextArea id={textareaId} onChange={onChange} placeholder="Check console for new value." />
```

Comment:
Another a11y thing (a nitpick, but we do want to model good practices in our examples) -- we shouldn't put any meaningful information in the placeholder, since it often doesn't meet contrast, and also disappears as soon as someone types.

I'd suggest having this instead:
```jsx
<Label htmlFor={textareaId}>Uncontrolled TextArea with placeholder.</Label>
<TextArea id={textareaId} aria-describedby="desc" onChange={onChange} placeholder="type here..." />
<span id="desc">Check console for new value.</span>
```

That way we both have more accessible examples, and also demo how to use `aria-describedby` as a bonus :D